### ISSUE TITLE
Vico charts - Add a content description

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#2799]
As discussed in #973:

> exposing the graph as an object with a description explaining it's a hourly graph. It would not solve the problem directly, but have two strong benefits:
> 
>     * People would know the values are there, even if not accessible, and could avoid looking for them elsewhere in the app
> 
>     * And TB has a gesture to send a component of the screen for analysis to Gemini, so they could get at least an idea of the graph shape, what could be a good start given the forecast reliability questions you mentioned

**Comment by papjul** [comment]
Will be supported upstream by patrykandpatrick/vico#1069

However, even when it will be supported, we can't update yet, because we need patrykandpatrick/vico#1407 to bring back image support in newer 3.x Vico versions
